index.js

```diff
@@ -3,14 +3,17 @@ const ManyKeysMap = require('many-keys-map');
 const pDefer = require('p-defer');
 
 const cache = new ManyKeysMap();
-const isDomReady = () => document.readyState === 'interactive' || document.readyState === 'complete';
+const isDomReady = target =>
+	(target.ownerDocument || target).readyState === 'interactive' ||
+	(target.ownerDocument || target).readyState === 'complete';
```

Comment:
I know it's nice with a one-liner, but how about:

```suggestion
const isDomReady = target => {
	const document_ = target.ownerDocument || target;

	return document_.readyState === 'interactive' ||
		document_.readyState === 'complete';
}
```

Reply:
I was thinking about:
```suggestion
const isDomReady = target =>
	['interactive', 'complete'].includes((target.ownerDocument || target).readyState);
```